src/test/java/LoginTests.java

```diff
@@ -15,7 +15,7 @@ public static void LoginEmptyEmailPasswordTest () {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://bbb.testpro.io/";
+        String url = "https://testpro.io/";
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void LoginEmptyEmailPasswordTest ()`